pkg/lib/infra/db/handle.go

```diff
@@ -13,5 +15,10 @@ type Handle interface {
 	ReadOnly(do func() error) (err error)
 
 	// conn allows internal access to the ongoing transaction.
-	conn() (sqlx.ExtContext, error)
+	conn() (*txConn, error)
+}
+
+type ConnContext interface {
```

Comment:
Is this used?

Reply:
This is actually the interface implemented by `txConn`; but since `conn()` is not public, it ends up unused. Let me remove this.